File: pkg/codegen/testing/test/testdata/transpiled_examples/random-pp/go/random.go
package main

import (
	"github.com/pulumi/pulumi-random/sdk/v4/go/random"
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
)

func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		randomPassword, err := random.NewRandomPassword(ctx, "randomPassword", &random.RandomPasswordArgs{
			Length:          pulumi.Int(16),
			Special:         pulumi.Bool(true),
			OverrideSpecial: pulumi.String("_%@"),
		})
		if err != nil {
			return err
		}
		ctx.Export("password", randomPassword.Result)
		return nil
	})
}

File: pkg/codegen/testing/test/testdata/transpiled_examples/random-pp/dotnet/random.cs
using System.Collections.Generic;
using System.Linq;
using Pulumi;
using Random = Pulumi.Random;

return await Deployment.RunAsync(() => 
{
    var randomPassword = new Random.RandomPassword("randomPassword", new()
    {
        Length = 16,
        Special = true,
        OverrideSpecial = "_%@",
    });

    return new Dictionary<string, object?>
    {
        ["password"] = randomPassword.Result,
    };
});

